app/scripts/contenteditor/migration/1_migration-task.js

```diff
@@ -82,5 +111,32 @@ org.ekstep.contenteditor.migration = new (Class.extend({
 	},
 	getBackupContent: function () {
 		return this.contentBackup
+	},
+	isAssessmentContent: function (contentbody) {
+		var instance = this
+		var assessmentContent = false
+		if (!_.isEmpty(instance.patch)) {
```

Comment:
@pallakartheekreddy - This check is redundant

Reply:
Yes, this is not required I'll remove.